blaze/expr/strings.py

```diff
@@ -54,7 +55,14 @@ def isstring(ds):
     return isinstance(getattr(measure, 'ty', measure), String)
 
 
+_add, _radd = _mkbin('add', Concat)
+_mod, _rmod = _mkbin('mod', Interp)
+_mul, _rmul = _mkbin('mul', Repeat)
+
+
 schema_method_list.extend([
+    (isstring, set([_add, _radd, _mod, _rmod, _mul, _rmul,
```

Comment:
@cpcloud The discovery is still blocking me on the pandas backend, I have no issue writing the compute_up functions; however, getting the methods to appear on the symbols is the issue.